de.html

```diff
@@ -1951,33 +1951,33 @@
 </p>
 
 <p>
-	🔊 This project was made with Creative Commons assets
-	from Wikimedia Commons and FreeSounds.
+	🔊 Dieses Projekt wurde mit Creative Commons Bestandteilen
```

Comment:
"Bestandteilen" gefällt mir nicht, mir fällt aber sonst nichts ein... Hat jemand eine Idee?

Reply:
Vllt "Beiträgen" oder "Informationen"?

Reply:
Ich finde "Beiträge" auch gut, also 
```suggestion
	🔊 Dieses Projekt wurde mit Creative Commons Beiträgen
```